components/ContractExecution.tsx

```diff
@@ -521,17 +521,16 @@ function ExecuteForm(
       <Formik
         enableReinitialize
         initialValues={props.shape.init}
-        onSubmit={async values => {
+        onSubmit={() => {}}
+        validate={values => {
           try {
             genLambda(props, values);
           } catch (e) {
-            console.log(e);
-            setSubmitError((e as Error).message);
-            props.setLoading(false);
+            // setSubmitError((e as Error).message);
```

Comment:
Well I'm not 100% sure of it. The reason I removed it is because it's redundant with the local error handling. So, if there's no error in the form above, it shouldn't throw right?

![image](https://user-images.githubusercontent.com/36401048/234282852-134770f5-0871-46d0-a5b9-e7415c0ad2e6.png)

Reply:
per offline discussion, comment `setSubmitError` for now.